### ISSUE TITLE
Failure in web interface of XSSP - KeyError: 'HG_HSSP_ROOT'

### ISSUE THREAD

**Issue by laurensvdwiel** [cmbi/xssp-api#80]
Failure on web interface of XSSP at http://www.cmbi.ru.nl/xssp/

With options: 
"Output": HSSP (Human Genome)
"Input": Sequence
with input sequence:
`MNGTEGPNFYVPFSNATGVVRSPFEYPQYYLAEPWQFSMLAAYMFLLIVLGFPINFLTLYVTVQHKKLRTPLNYILLNLAVADLFMVLGGFTSTLYTSLHGYFVFGPTGCNLEGFFATLGGEIALWSLVVLAIERYVVVCKPMSNFRFGENHAIMGVAFTWVMALACAAPPLAGWSRYIPEGLQCSCGIDYYTLKPEVNNESFVIYMFVVHFTIPMIIIFFCYGQLVFTVKEAAAQQQESATTQKAEKEVTRMVIIMVIAFLICWVPYASVAFYIFTHQGSNFGPIFMTIPAFFAKSAAIYNPVIYIMMNKQFRNCMLTTICCGKNPLGDDEASATVSKTETSQVAPA`

Result:

```
Traceback (most recent call last):
  File "/srv/www/xssp-rest/.virtualenvs/xssp-rest/local/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/srv/www/xssp-rest/xssp-rest/xssp_rest/factory.py", line 133, in __call__
    return TaskBase.__call__(self, *args, **kwargs)
  File "/srv/www/xssp-rest/.virtualenvs/xssp-rest/local/lib/python2.7/site-packages/celery/app/trace.py", line 437, in __protected_call__
    return self.run(*args, **kwargs)
  File "/srv/www/xssp-rest/xssp-rest/xssp_rest/tasks.py", line 179, in get_hg_hssp
    path = os.path.join(flask_app.config['HG_HSSP_ROOT'],
KeyError: 'HG_HSSP_ROOT'
```

Job output: http://www.cmbi.ru.nl/xssp/output/sequence/hg_hssp/c4c7e7ce-3665-44f9-a6f5-f41175f28b80

**Comment by jonblack** [comment]
The `HG_HSSP_ROOT` setting was added to `dev_settings.py` but not updated on the production server.

I've updated it on the server but this issue should remain open so it can be fixed more permanently.

**Comment by cbaakman** [comment]
I see that this setting has been moved to ```xssp_api/default_settings.py```. Does this mean we can close the issue?

**Comment by jonblack** [comment]
Correct. It was fixed in #86.
